aspnetcore/publishing/iis.md

```diff
@@ -144,6 +144,8 @@ If you don't have a *web.config* file in the project when you publish with *dotn
 ## Deploy the application
 Deploy the application to the folder you created on the target IIS system. Web Deploy is the recommended mechanism for deployment. Alternatives to Web Deploy are listed below.
 
+Confirm that the published app for deployment isn't running. Files in the *publish* folder are locked when the app is running, and deployment can't occur when files are locked because locked files can't be copied.
```

Comment:
Confirm that the published app for deployment isn't running. Files in the *publish* folder are locked when the app is running,  Deployment can't occur  because locked files can't be copied.